### PR TITLE
PV2-252-reenable-acceptance-test

### PR DESCRIPTION
**Pull request by hardik-desai-kcl**
…cted

### CODE REVIEW

**Review by AkshayBanifatsyevichChugh — DISMISSED** [review]

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Inconsistent Submission Data/PV2-288 Levy learner, start date moved forward, no payments refunded.feature [hunk 1]
```diff
@@ -1,15 +1,13 @@
-﻿@ignore
-#issue with maths and english
-Feature: Levy learner moves start date forward payments refunded including english & maths - PV2-288
+﻿Feature: Levy learner moves start date forward payments refunded including english & maths - PV2-288
 		As a levy employer,
-		I want a learner that moves start date forward no prog payments and English & Maths refunded, to be paid correct amount,
+		I want a learner that moves start date forward on prog payments and English & Maths refunded, to be paid correct amount,
```
Comment (tom-gough) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Inconsistent Submission Data/PV2-288 Levy learner, start date moved forward, no payments refunded.feature:3]: not super clear what this is trying to say, perhaps "I want a learner that moves start date forward, with on prog and English & Maths payments that have been refunded, to be paid correct amount"

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-290 - Levy learner changes standard with change to price.feature [hunk 2]
```diff
@@ -10,14 +8,14 @@ Scenario Outline: Levy learner change to standard at the end of a month along wi
...
+		| Apprenticeship 2 | 01/Aug/Current Academic Year | 01/Aug/Next Academic Year | 5625         | active | 03/Nov/Current Academic Year |                              | 25             | 52            |
 
 	And the following aims
-		| Aim Reference | Start Date                   | Planned Duration | Actual Duration | Aim Sequence Number | Programme Type | Standard Code | Funding Line Type             | Completion Status |
-		| ZPROG001      | 03/Aug/Current Academic Year | 12 months        | 3 months        | 1                   | 25             | 51            | 16-18 Apprenticeship Non-Levy | withdrawn         |
-		| ZPROG001      | 03/Nov/Current Academic Year | 9 months         |                 | 2                   | 25             | 52            | 16-18 Apprenticeship Non-Levy | continuing        |
+		| Aim Type         | Aim Reference | Start Date                   | Planned Duration | Actual Duration | Aim Sequence Number | Programme Type | Standard Code | Funding Line Type             | Completion Status |
```
Comment (tom-gough) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-290 - Levy learner changes standard with change to price.feature:16]: what is aim type? I can kind of guess that it's something added to represent standards vs frameworks, but we don't represent it this way anywhere else and generally I thought it was referred to as standard or framework, not programme, and a programme type is just something you need as part of a standard?

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-296 - Levy learner goes on a planned break.feature [hunk 3]
```diff
@@ -1,127 +1,130 @@
...
+        | R06/Current Academic Year |
+        | R07/Current Academic Year |
+        | R08/Current Academic Year |
+        | R09/Current Academic Year |
+        | R10/Current Academic Year |
+        | R11/Current Academic Year |
+        | R12/Current Academic Year |
+
+
+ #Scenario: Apprentice goes on a planned break midway through the learning episode and this is notified through the ILR
```
Comment (tom-gough) [inline src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-296 - Levy learner goes on a planned break.feature:115]: I get that this is existing commented out code, but do we definitely still want it?

**Review by tom-gough — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-290 - Levy learner changes standard with change to price.feature (hunk 2 above)
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-290 - Levy learner changes standard with change to price.feature:16]: | Aim Type             |
| Maths or English |
| Programme         |

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Inconsistent Submission Data/PV2-288 Levy learner, start date moved forward, no payments refunded.feature (hunk 1 above)
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Inconsistent Submission Data/PV2-288 Levy learner, start date moved forward, no payments refunded.feature]: fixed

File: src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-296 - Levy learner goes on a planned break.feature (hunk 3 above)
Comment (hardik-desai-kcl) [reply src/SFA.DAS.Payments.AcceptanceTests.EndToEnd/Tests/Learner Change of Circumstances/PV2-296 - Levy learner goes on a planned break.feature:115]: this is the old style acceptance tests

**Review by tom-gough — DISMISSED** [review]
Checking tests on my machine, all changed tests passing

**Review by AkshayBanifatsyevichChugh — APPROVED** [review]

**Pull request merged**